app/Commands/Dev/Anoma/Prove.hs

```diff
@@ -0,0 +1,27 @@
+module Commands.Dev.Anoma.Prove where
+
+import Anoma.Effect.Base
+import Anoma.Effect.RunNockma
+import Commands.Base
+import Commands.Dev.Anoma.Base
+import Commands.Dev.Anoma.Prove.Options
+import Juvix.Compiler.Nockma.Encoding.Jam qualified as Encoding
+import Juvix.Compiler.Nockma.Pretty hiding (Path)
+
+runCommand :: forall r. (Members (Anoma ': Error SimpleError ': AppEffects) r) => ProveOptions -> Sem r ()
+runCommand opts = do
+  res <- runNock (opts ^. proveFile) (opts ^. proveArgs)
+  let traces = res ^. runNockmaTraces
+  forM_ traces (renderStdOutLn . ppPrint)
```

Comment:
We should be using ppOutDefault instead of ppPrint so that we see colors in the output
```suggestion
  forM_ traces (renderStdOutLn . ppOutDefault)
```